index.html

```diff
@@ -26,6 +26,10 @@ <h1>Welcome to the Scrimba Bootcamp 2024!</h1>
             <img src="images/coffee.jpg" alt="A cup of coffee seen from above and with coffee beans around it" />
         </div>
         
+        <div id="feed">
```

Comment:
We could potentially use the semantic `<section>` element here rather than a generic `<div>`.

Reply:
Yes, you are right Micha! I'll create some new issues for those changes